scripts/run-in-pipeline.sh

```diff
@@ -0,0 +1,194 @@
+##
+#  This script gets packaged with a Redhat python UBI (Universal Base Image) and serves as the entrypoint of the container.
+#  The IBM Detect Secrets CLI tool is also packaged in the container; these scripts will run:
+#      1) detect-secrets scan --update
+#      2) followed by:  detect-secrets audit --report with fail-on options
+#  to generate a report and emit an exit code (1 = fail, 0 = pass) on selected conditions based on the specified fail-on flags.
+#
+#  Note:  We include a SKIP_SCAN flag if one desired to skip the scan --update part (step 1 above) and just to the audit -report with existing baseline
+#         This option might be chosen in the case that the user has already updated their baseline file locally
+#
+#  This script takes the following parameters via Environment Variables:
+#    BASELINE :  This is the filename to use to store/read the scan baseline output/input.
+#      * Default:  .secrets.baseline
+#
+#    SKIP_SCAN : (True/False) Allows for the skipping of the Scan --update part that updates the baseline file.
+#      * Default:  False
+#
+#    JSON : (True/False) Whether to output result in JSON format vs the Table format
+#      * Default:  False
+#        Note:  Setting True is mutually exclusive with setting OMIT_INSTRUCTIONS=True.
+#
+#    OMIT_INSTRUCTIONS : (True/False) Whether to output resolution instructions for failed conditions.
+#      * Default:  False
+#        Note:  Setting True is mutually exclusive with setting JSON=True.
+#
+#    FAIL_ON_LIVE : (True/False) Sets the condition to fail audit if there are verified live secrets found
+#      * Default:  True
+#
+#    FAIL_ON_UNAUDITED : (True/False) Sets the condition to fail audit if there are unaudited potential secrets found
+#      * Default:  True
+#
+#    FAIL_ON_AUDITED_REAL : (True/False) Sets the condition to fail audit if there are audited and set to real secrets found
+#      * Default:  True
+#
+# Example manual docker run within packaged container (git-defenders/detect-secrets-ubi):
+#    docker run  --env BASELINE=.secrets.baseline  --env FAIL_ON_LIVE=False  -it -a stdout --rm -v $(pwd):/code git-defenders/detect-secrets-ubi
+##
+
+## Constants for FAIL_ON_xx Environment Varibles
+_false="false"
+_true="true"
+
+## Setting Baseline filename, skip_scan, and fail-on-xx Boolean Defaults
+_baseline_default=.secrests.baseline
+_skip_scan_default=$_false
+_json_default=$_false
+_omit_instructions_default=$_false
+_fail_live_default=$_true
+_fail_unaudited_default=$_true
+_fail_audited_real_default=$_true
+
+## Constants representing Detect Secrets audit -reports --fail-on-xx paramenter options
+_fail_live_option="--fail-on-live"
+_fail_unaudited_option="--fail-on-unaudited"
+_fail_audited_real_option="--fail-on-audited-real"
+_omit_instructions_option="--omit-instructions"
+_json_option="--json"
+
+## Reading input Environment Variables while setting defaults for missing Environment Variables
+baseline=${BASELINE:=.$_baseline_default}
```

Comment:
This var isn't used, it looks like `$BASELINE` is getting referenced instead of `$baseline`
<img width="663" alt="Screen Shot 2022-03-18 at 10 49 17 AM" src="https://user-images.githubusercontent.com/12804086/159025524-025f6c9a-6172-471c-aca6-936393423497.png">
[VS Code plugin](https://marketplace.visualstudio.com/items?itemName=timonwong.shellcheck)

Reply:
I need to change it to use $baseline - good caught.

Reply:
the 4 occuruces of $BASELINE should be $baseline -- changing now